.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: required
```

Comment:
Have no fear, travis is here 
<img width="779" alt="screen shot 2016-07-14 at 18 45 35" src="https://cloud.githubusercontent.com/assets/777823/16847829/2d9fd768-49f3-11e6-856c-bbc97ff501e9.png">